python/openai/README.md

```diff
@@ -315,3 +315,15 @@ available arguments and default values.
 
 For more information on the `tritonfrontend` python bindings, see the docs
 [here](https://github.com/triton-inference-server/server/blob/main/docs/customization_guide/tritonfrontend.md).
+
+## Known Limitations
+
+- For multi-GPU models (ex: Tensor Parallelism > 1), here is the current support matrix:
+  - [x] vLLM ([EngineArgs](https://github.com/triton-inference-server/vllm_backend/blob/main/README.md#using-the-vllm-backend))
+    - ex: Configure `tensor_parallel_size: 2` in the
+      [model.json](https://github.com/triton-inference-server/vllm_backend/blob/main/samples/model_repository/vllm_model/1/model.json)
+  - [x] TensorRT-LLM ([Orchestrator Mode](https://github.com/triton-inference-server/tensorrtllm_backend/blob/main/README.md#orchestrator-mode))
+    - `export TRTLLM_ORCHESTRATOR=1`
+    - **NOTE**: Orchestrator mode does not currently support multi-node models
+  - [ ] TensorRT-LLM ([Leader Mode](https://github.com/triton-inference-server/tensorrtllm_backend/blob/main/README.md#leader-mode))
```

Comment:
this is a mode and not a model parellism thing .... not sure best way - maybe a section on model parallelism and a seperate known limitations or a subsection in

Reply:
These are currently the different ways you can do model parallelism with Triton's LLM backends and what you need to be aware of when thinking about how you're going to achieve it with Triton. 

I think it's relatively clear in this current structure as an eye chart, with links to the details of each (vllm backend documentation on parallelism is currently a bit light), and adds some discover-ability into the methods.

I don't think the OpenAI README needs to go into significantly deeper depth on the concepts other than linking to the relevant sections - but let me know if you want to see something specific added

Reply:
I've got some thoughts but would be bigger than this change - so will take them offline.